Delivery-Service.API/Controllers/WeatherForecastController.cs

```diff
@@ -0,0 +1,54 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using DeliveryService.Contracts;
+using DeliveryService.Database;
+using DeliveryService.Services;
+using DeliveryService.Models;
```

Comment:
add build check in GitHub
close PRs from other modules
remove this default "WeatherForecastController "

Reply:
create your own
or rename it

Reply:
how to add build check?